types/stack/branch/compare.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Response } from '../../contentstackCollection';
+import { AnyProperty } from '../../utility/fields';
+import { Pagination } from '../../utility/pagination';
+
+export interface Compare {
+    all(params: Pagination & AnyProperty): Promise<AnyProperty>
+    contentTypes(params: Params) : Promise<Response>
```

Comment:
@nadeem-cs update return type from `Promise<Response>` to `Promise<AnyProperty>` for `contentTypes` and `globalFields` function